doc/dynamic_recomposition.md

```diff
@@ -0,0 +1,60 @@
+# Dynamic Recomposition
+
+VS-MEF does not support .NET MEF's dynamic recomposition feature. This feature is exposed in .NET MEF in several ways, including:
```

Comment:
What's going to happen with ImportAttribute.AllowRecomposition property? Is it going to be ignored?

Reply:
Yes, we ignore that parameter. it's fine if they support it -- we just don't ever do it.